### ISSUE TITLE
Proposed TC with saved content is changed to Confirmed TC with no content if TP with no Confirmed TCs is used to change status of TCs to Confirmed

### ISSUE THREAD

**Issue by ronfeldman** [kiwitcms/Kiwi#1320]
### Description of problem
TC which was saved with content and was in the PROPOSED state is changed to a TC with no content in the CONFIRMED state if, from Test Plan view -> Cases tab -> Status -> Set CONFIRMED is clicked

### Version or commit hash (if applicable)
7.3

### Steps to Reproduce

1. Create and save TP1
2. Create TC1 for TP1
3. Save TC1 with content. Do not change Status (i.e. save it in PROPOSED Status)
4. View TP1
5. Select Cases tab. No TCs are listed.
6. Click on Status -> Set Confirmed

### Actual results
TC with no summary and Status CONFIRMED is listed. When this TC is viewed by clicking on its ID, it is empty.
Reviewing Cases tab shows no TCs

### Expected results
No TCs are changed. OR TC which was in PROPOSED State is changed to CONFIRMED State and all content is retained.

### Additional info (Python traceback, logs, etc.)

Tried to duplicate this in https://public.tenant.kiwitcms.org but it expected results were observed.

Attaching images to show screens at different stages of steps to recreate.
![pic1](https://user-images.githubusercontent.com/6746868/72944591-2bbfc700-3d2e-11ea-936c-30944707407c.jpg)
![pic2](https://user-images.githubusercontent.com/6746868/72944596-2f534e00-3d2e-11ea-8e8d-7fe5c1005283.jpg)
![pic3](https://user-images.githubusercontent.com/6746868/72944601-34b09880-3d2e-11ea-9212-86ca37e02a0b.jpg)
![pic4](https://user-images.githubusercontent.com/6746868/72944605-37ab8900-3d2e-11ea-8415-1d24cb343075.jpg)
![pic5](https://user-images.githubusercontent.com/6746868/72944609-39754c80-3d2e-11ea-89b2-c20e6daab208.jpg)

**Comment by atodorov** [comment]
Duplicate of https://github.com/kiwitcms/Kiwi/issues/1318

The fix will be released in the next version.

**Comment by ronfeldman** [comment]
> Duplicate of #1318
> 
> The fix will be released in the next version.

Great that it was fixed. Out of concern that users will accidentally delete content, I've patched templates/plan/get_cases.html, commenting out the section with toolbar_status and toolbar_priority in the meantime.

I'd like to see high priority defects like this treated with an official patch or something similar. I'm starting a slow roll-out to devs and others, and I wouldn't want their curiosity to lead to lost QA work, and then for that to cause difficulties in gaining acceptance of the tool.
